back/src/main/resources/properties/resilience.yml

```
@@ -0,0 +1,58 @@
+        failureRateThreshold: 50         # 실패율 50% 이상이면 OPEN
+        waitDurationInOpenState: 5s      # OPEN 유지 시간
+        permittedNumberOfCallsInHalfOpenState: 5
+        automaticTransitionFromOpenToHalfOpenEnabled: true
+        recordExceptions:
+          - org.springframework.dao.DeadlockLoserDataAccessException
+          - org.springframework.dao.CannotAcquireLockException
+          - org.springframework.dao.TransientDataAccessException
+          - java.sql.SQLTransientException
+          - java.sql.SQLRecoverableException
+        ignoreExceptions:
+          - java.lang.IllegalArgumentException # 재시도해도 소용없는 것들
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)